src/components/creator/StatementEdition.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Switch, TextField } from "@mui/material";
+import { useState } from "react";
+import { LocalStorage } from "../../localStorage";
+import { useTranslation } from "react-i18next";
+import { GenericModalDialog } from "../modalDialog/GenericModalDialog";
+
+export const StatementEdition = () => {
+
+    const { t } = useTranslation('creator');
+
+    const currentStatement = LocalStorage.getCreatorChallenge()!.statement.description
+    const currentClue = LocalStorage.getCreatorChallenge()!.statement.clue
+
+    const [statementInProgress, setStatementInProgress] = useState(currentStatement);
+    const [clueInProgress, setClueInProgress] = useState(currentClue);
+    const [clueCheck, setClueCheck] = useState(currentClue !== '' && currentClue !== undefined);
```

Comment:
```suggestion
    const [clueCheck, setClueCheck] = useState(!!currentClue)
```
Una boludez, acá justo la idea de "truthyness" de js creo que ayuda a hacer esto más directo.

Reply:
no lo recordaba. Es cierto.